.travis.yml

```diff
@@ -9,6 +9,7 @@ python:
   - "2.7"
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
Seems like 3.7 is not officially supported on Travis yet. You might need to do (https://github.com/travis-ci/travis-ci/issues/9815)

Reply:
Ah thanks!